src/components/leaderboard/ducks/types.ts

```diff
@@ -0,0 +1,17 @@
+import { C4CState } from '../../../store';
+import { ThunkAction } from 'redux-thunk';
+import { LeaderboardItemAction } from './actions';
+import { ApiExtraArgs } from '../../../api/apiClient';
+
+export interface LeaderboardItem {
+  id: number;
+  name: string;
+  blocksCounted: number;
+}
+
+export type LeaderboardThunkAction<R> = ThunkAction<
```

Comment:
I think this should also be in a `types` file in each of the ducks folders as opposed to one location